### PR TITLE
NX-33 - Part 1

### PR DESCRIPTION
**Pull request by cor**
- Cleans up our repository's flake-parts structure
- Removes a ton of old junk
  - Please review carefully, I'm quite sure that I didn't delete anything important, but I may have missed something.
- Cleanup isn't final, still more TBD

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [NX-33 Cleanup of the flake-parts structure](https://app.clickup.com/t/20465559/NX-33)

**Comment by vimukthi-git** [comment]
sir is there any chance that things could break for v2.10004 testing and release with this in the coming days? If so I recommend delaying refactorings to build process to once v2 runtime upgrade is done.

**Comment by dzmitry-lahoda** [comment]
yeah, and hopefully after zn release. also, i do not thing that 2 points from plan are good. and another one co trovercial.

**Review by dzmitry-lahoda — CHANGES_REQUESTED** [review]
please align changes in time and space. i mean for releases and that changes are proper(not waste)

**Comment by KaiserKarel** [comment]
> sir is there any chance that things could break for v2.10004 testing and release with this in the coming days? If so I recommend delaying refactorings to build process to once v2 runtime upgrade is done.

It shouldn't, as this just moves existing files around, but @cor we can keep this in draft until tuesday to avoid any issues. :)

**Comment by cor** [comment]
The hashes of the outputs remain identical, but if this scares people then sure lets wait with merging it 👍

**Review by dzmitry-lahoda — APPROVED** [review]

File: inputs/paritytech/zombienet.nix [hunk 1]
```diff
@@ -8,16 +8,28 @@
...
           src = "${paritytech-zombienet-src}/javascript";
           buildInputs = with pkgs; [ nodejs ];
-          nativeBuildInputs = npmDeps.nativeBuildInputs;
+          nativeBuildInputs = with pkgs; [
+            yarn
+            nodejs
+            python3
+            pkg-config
+            vips
+            python3
```
Comment (obsessed-cake) [inline inputs/paritytech/zombienet.nix:27]:
```suggestion
            python3
            pkg-config
            vips
```

**Review by obsessed-cake — APPROVED** [review]

**Review by KaiserKarel — APPROVED** [review]

**Review by Dom-Roth — APPROVED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]

**Pull request merged**